opencti-platform/opencti-graphql/src/manager/taskManager.js

```diff
@@ -497,6 +504,56 @@ const sharingOperationCallback = async (context, user, task, actionType, operati
   };
 };
 
+const ruleApplyCallback = async (context, user, task, ruleId) => {
+  return async (elements) => {
+    for (let index = 0; index < elements.length; index += 1) {
+      const inferredObjectsBundle = [];
+      const element = elements[index];
+      // in case of rule apply we need to fake the apply of the rule and get the resulting inferred creations
+      const inferredRelations = [];
+      const inferredEntities = [];
+      const createInferredRelationOverride = (_context, input, ruleContent, opts) => {
+        inferredRelations.push({ input, ruleContent, opts });
```

Comment:
Accumulate everything before sending bundle could cause on OOM. Can you change that to send bundle to queue incrementaly ?